docs/root/development/releasing/releasing.rst

```diff
@@ -0,0 +1,40 @@
+.. _releasing_instructions:
+
+Releasing
+=========
+
+Prepare for release
+-------------------
+
+The following workflow should be followed to create and publish a new Envoy Mobile
```

Comment:
sorry, this is silly, but could you put this sentence between Releasing and prepare for release? It applies to both subsections.

Reply:
done.